build/dockerfiles/Dockerfile

```diff
@@ -19,6 +19,13 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+FROM docker.io/node:12.20.1-alpine3.12 as dwtemplates
+RUN apk add --no-cache bash && npm install -g @eclipse-che/che-theia-devworkspace-handler
```

Comment:
the version of the library @eclipse-che/che-theia-devworkspace-handler should be added to be sure to have a reproducible build

Reply:
done